README.mkd

```diff
@@ -525,6 +525,23 @@ So, to avoid any trouble, run `antigen use oh-my-zsh` if you haven't already
 before experimenting with themes. If you have `antigen use oh-my-zsh` in your
 `.zshrc`, you're covered.
 
+**Note**: Antigen compiles bundles to avoid loading them each time a terminal launches. That behaviour can leads to a side effect when a theme isn't loading when you try to do so from your .zshrc. For example: 
```

Comment:
> Antigen compiles bundles to avoid...

This would be my approach. Although I think this probably belongs to the Wiki. A shorter description would be more appropriated for the `README.md`:

Antigen caches bundles in order to provide fast start up times. This functionality may cause issues if you're dynamically loading bundles based on environment variables or other conditionals. For example:

    case $(uname) in
    Darwin)
        antigen theme pure
    ;;
    *)
        antigen theme agnoster
    ;;
    esac

In this scenario you have a couple of options:

   - You may disable caching with `ANTIGEN_CACHE=false`. This will affect start up time when you've more than 10 bundles.

   - You may use cache for the different environments with `ANTIGEN_CACHE=$HOME/.antigen/init-$(uname).zsh`